2024-02-27 19:49:32,287 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Configure stats pid to 20269
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Loading settings from /home/sadbhawna/.config/wandb/settings
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Loading settings from /home/sadbhawna/Desktop/cs6910_assignment1/wandb/settings
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'cs23d014_assignment_1', 'entity': 'cs23d014', 'root_dir': '/home/sadbhawna/Desktop/cs6910_assignment1', 'run_id': 'sjtrvmg5', 'sweep_param_path': '/home/sadbhawna/Desktop/cs6910_assignment1/wandb/sweep-bcazvlsh/config-sjtrvmg5.yaml', 'sweep_id': 'bcazvlsh'}
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-27 19:49:32,288 INFO    Thread-20 (_run_job):20269 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'sweep_functionality.py', 'program_abspath': '/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py', 'program': '/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py'}
2024-02-27 19:49:32,289 INFO    Thread-20 (_run_job):20269 [wandb_init.py:_log_setup():526] Logging user logs to /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240227_194932-sjtrvmg5/logs/debug.log
2024-02-27 19:49:32,289 INFO    Thread-20 (_run_job):20269 [wandb_init.py:_log_setup():527] Logging internal logs to /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240227_194932-sjtrvmg5/logs/debug-internal.log
2024-02-27 19:49:32,289 INFO    Thread-20 (_run_job):20269 [wandb_init.py:init():566] calling init triggers
2024-02-27 19:49:32,289 INFO    Thread-20 (_run_job):20269 [wandb_init.py:init():573] wandb.init called with sweep_config: {'activation': 'tanh', 'batch_size': 16, 'dataset': 'fashion_mnist', 'epochs': 10, 'eps': 0.0001, 'hidden_size': 32, 'learning_rate': 0.001, 'loss': 'mean_squared_error', 'num_layers': 5, 'optimizer': 'sgd', 'wandb_project': 'cs23d014_assignment_1', 'weight_decay': 0.5, 'weight_ini': 'Xavier Normal'}
config: {}
2024-02-27 19:49:32,290 INFO    Thread-20 (_run_job):20269 [wandb_init.py:init():611] wandb.init() called when a run is still active
2024-02-27 19:49:34,592 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_finish():1970] finishing run cs23d014/cs23d014_assignment_1/sjtrvmg5
2024-02-27 19:49:34,593 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 1
2024-02-27 19:49:34,593 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_restore():2193] restore
2024-02-27 19:49:34,596 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_restore():2199] restore done
2024-02-27 19:49:40,569 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-02-27 19:49:40,569 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-02-27 19:49:40,572 INFO    Thread-20 (_run_job):20269 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-02-27 19:49:40,688 ERROR   MainThread:20269 [pyagent.py:_run_jobs_from_queue():236] Run sjtrvmg5 errored:
Traceback (most recent call last):
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/wandb/agents/pyagent.py", line 308, in _run_job
    self._function()
  File "/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py", line 92, in train
    fit(layer_architecture, X_train, Y_train, X_val, Y_val, X_test, Y_test,
  File "/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py", line 45, in fit
    SGD(layer_architecture, X_train, Y_train, X_val, Y_val, X_test, Y_test, epochs=epochs, activation = activation,loss=loss, weight_ini = weight_ini, learning_rate=learning_rate, batch_size=batch_size, weight_decay=weight_decay, project="cs23d014_assignment_1")
  File "/home/sadbhawna/Desktop/cs6910_assignment1/optimizers.py", line 73, in SGD
    wandb.log({'confusion_matrix': wandb.plot.confusion_matrix(probs=None, y_true=y_true_class_indx,preds=preds_class_indx,class_names=labels)})
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/wandb/plot/confusion_matrix.py", line 95, in confusion_matrix
    wandb.Table(columns=["Actual", "Predicted", "nPredictions"], data=data),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/wandb/data_types.py", line 215, in __init__
    elif util.is_pandas_data_frame(data):
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/wandb/util.py", line 449, in is_pandas_data_frame
    import pandas as pd
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/__init__.py", line 48, in <module>
    from pandas.core.api import (
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/api.py", line 47, in <module>
    from pandas.core.groupby import (
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/groupby/__init__.py", line 1, in <module>
    from pandas.core.groupby.generic import (
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/groupby/generic.py", line 77, in <module>
    from pandas.core.frame import DataFrame
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/frame.py", line 182, in <module>
    from pandas.core.generic import NDFrame
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/generic.py", line 179, in <module>
    from pandas.core.window import (
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/window/__init__.py", line 1, in <module>
    from pandas.core.window.ewm import (
  File "/home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/core/window/ewm.py", line 11, in <module>
    import pandas._libs.window.aggregations as window_aggregations
ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /home/sadbhawna/anaconda3/lib/python3.11/site-packages/pandas/_libs/window/aggregations.cpython-311-x86_64-linux-gnu.so)

2024-02-27 19:49:40,689 ERROR   MainThread:20269 [pyagent.py:_run_jobs_from_queue():261] Detected 5 failed runs in a row at start, killing sweep.
